python/fledge/common/web/ssl_wrapper.py

```diff
@@ -80,21 +80,27 @@ def verify_against_revoked(cls):
     @classmethod
     def verify_against_ca(cls):
         echo_process = subprocess.Popen(['echo', cls.user_cert], stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-        a = subprocess.Popen(["openssl", "verify", "-CAfile", cls.ca_cert, "-x509_strict"], stdin=echo_process.stdout, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+        args = ["openssl", "verify", "-CAfile", cls.ca_cert]
+        cmd = "openssl verify -CAfile {}".format(cls.ca_cert)
```

Comment:
We can avoid this and use ' '.join(args) where required

Reply:
When joining strings Python only has to allocate new memory for the final string, which is much more efficient, but could take longer to compute.
For example foo += 'bar' is just as readable (according to me) and more faster than foo = ''.join([foo, 'bar']).